kani-compiler/src/kani_middle/transform/check_aliasing/local_collection.rs

```diff
@@ -0,0 +1,64 @@
+use super::function_cache::*;
+use super::{
+    Body, BodyMutationPassState, CachedBodyMutator, InstrumentationData, Local, LocalDecl,
+    MirVisitor, TyCtxt,
+};
+use std::collections::HashMap;
+
+/// Collect local visitor visits the body
+/// and collects all of the (non argument)
+/// locals. In the future it will collect
+/// argument locals and initialize arguments.
+struct CollectLocalVisitor {
```

Comment:
Similarly to the previous comments, I don't think you need a new struct for that.